docs/manifest.json

```diff
@@ -258,16 +258,14 @@
 				},
 				{
 					"title": "Coder Desktop",
-					"description": "Use Coder Desktop to access your workspace like it's a local machine",
+					"description": "Transform remote workspaces into seamless local development environments with no port forwarding required",
```

Comment:
we can try it, but it's not accurate - the remote workspace is not affected by your use of Desktop.

Reply:
Keeping it now.